Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/TimeSynchronizer.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    internal class TimeSynchronizer : Singleton<TimeSynchronizer>
```

Comment:
>internal class TimeSynchronizer : Singleton<TimeSynchronizer> [](start = 4, length = 61)

The usage of this class from HologramSynchronizer should in some way be replaced with CompositionManager.GetHologramTime, which will do the correct time adjustment when compositing from a DSLR camera rig.